references.bib

```diff
@@ -0,0 +1,168 @@
+
+@misc{monolith2017shadow,
+  title={Middle-earth: Shadow of War},
+  publisher={Waner Bros. Interactive Entertainment},
+  organization={Monolith Productions},
+  year={2017}
+}
+
+
+@misc{ubisoft2013assassins,
+  title={Assassin's Creed IV: Black Flag},
+  publisher={Ubisoft},
+  organization={Ubisoft},
+  year={2013}
+}
+
+@inproceedings{hauge2014implications,
+  title={Implications of learning analytics for serious game design},
+  author={Hauge, Jannicke Baalsrud and Berta, Riccardo and Fiucci, Giusy and Manj{\'o}n, Baltasar Fern{\'a}ndez and Padr{\'o}n-N{\'a}poles, Carmen and Westra, Wim and Nadolski, Rob},
+  booktitle={Advanced Learning Technologies (ICALT), 2014 IEEE 14th International Conference on},
+  pages={230--232},
+  year={2014},
+  organization={IEEE}
+}
+
+@inproceedings{cowling2015emergent,
+  title={Emergent bluffing and inference with Monte Carlo Tree Search},
+  author={Cowling, Peter I and Whitehouse, Daniel and Powley, Edward J},
+  booktitle={Computational Intelligence and Games (CIG), 2015 IEEE Conference on},
+  pages={114--121},
+  year={2015},
+  organization={IEEE}
+}
+
+@article{game2012accessability,
+  title={Accessibility Validators},
+  author={Barrie, Ellis and Gareth, Ford-Williams and Lynsey, Graham and Dimitris, Grammenos and Ian, Hamilton and Ed, Lee and Jake, Manion and Thomas, Westin},
+  url={http://gameaccessibilityguidelines.com/},
+  year = {2012},
+  urldate = {10/03/2017}
+}
+
+@article{bergasa2005intelligent,
+  title={Intelligent browser-based systems to assist Internet users},
+  author={Bergasa-Suso, Jorge and Sanders, David A and Tewkesbury, Giles E},
+  journal={IEEE Transactions on Education},
+  volume={48},
+  number={4},
+  pages={580--585},
+  year={2005},
+  publisher={IEEE}
+}
+
+@article{gervas2009storytelling,
+  title={to Storytelling and Creativity},
+  author={Gerv{\'a}s, Pablo},
+  year={2009}
+}
+
+@article{klein1973automatic,
+  title={Automatic novel writing: A status report},
+  author={Klein, Sheldon and Aeschlimann, J and Balsiger, D and others},
+  journal={Wisconsin University},
+  year={1973}
+}
+
+@inproceedings{do2016trust,
+  title={A Trust and Reputation Framework for Game Agents: Providing a Social Bias to Computer Players},
+  author={do Couto, Fabio S and Delgado, Carla ADM and da Silva, Jo{\~a}o CP},
+  booktitle={Intelligent Systems (BRACIS), 2016 5th Brazilian Conference on},
+  pages={193--198},
+  year={2016},
+  organization={IEEE}
+}
+
+@article{akimotoexploratory,
+  title={Exploratory Approach to the Computational Modeling of Narrative Ability for Artificial Intelligence},
+  booktitle={International Journal of Knowledge Engineering},
+  volume={Vol. 2},
+  number = {No. 4},
+  year = {2016},
+  author={Akimoto, Taisuke}
+}
+
+@article{bringsjord2000artificial,
+  title={Artificial intelligence and literary creativity: Inside the mind of BRUTUS, a storytelling machine},
```

Comment:
Also needs extra curly braces for capitalisation. Note that you can just put them around the word(s) that BibTeX shouldn't mess with, i.e.
```
title={Artificial intelligence and literary creativity: Inside the mind of {BRUTUS}, a storytelling machine}
```